### ISSUE TITLE
我觉得这有些问题

### ISSUE THREAD

**Issue by xingran521** [Yourdaylight/MiniProgramCloudDevManage#1]
1. 你的库环境还需增加
      PyMySQL            1.0.3
      Flask              2.2.3
      Jinja2             3.1.2
      这些是目前我看到的环境以及版本
2. 你的项目还有好多空的文件没有补全
3. 你的项目生成头部注释没有写明文件功能以及每个类和函数没有写明用法
4. 你的开发还没有完成，我是通过NEW Bing 找到你的项目的

如还有什么问题可以支持一起探讨

**Comment by xingran521** [comment]
Of course, if you are not Chinese and can't read the above, here is the English version

1. Your library environment also needs to add
      PyMySQL 1.0.3
      Flask 2.2.3
      Jinja2 3.1.2
      These are the environments and versions I've seen so far
2. Your project still has a lot of empty files that are not filled in
3. Your project generates header comments that do not specify the function of the file and the usage of each class and function
4. Your development is not yet complete, I found your project through NEW Bing

If you have any questions, please support us to discuss

**Comment by Yourdaylight** [comment]
项目新建，还没开始编码

**Comment by xingran521** [comment]
什么时候开始编码呢，是否帮忙？

**Comment by Yourdaylight** [comment]
谢谢啦，暂时不用，过两周再开始，最近没啥时间
